sbtc-core/src/signer/coordinator/mod.rs

```diff
@@ -25,8 +27,39 @@ pub struct PublicKeys {
 /// TODO: Define the Message types for Tx Signning Round
 /// https://github.com/stacks-network/sbtc/issues/43
 
+/// An sBTC transaction needing to be processed by the coordinator
+/// TODO: replace with the core library's SBTCTransaction
+/// This could be a BTC transaction or a STX transaction
+/// depending on https://github.com/Trust-Machines/stacks-sbtc/pull/595
+pub enum SBTCTransaction {
+    /// A commit Bitcoin transaction
+    Commit(TaprootSpendInfo, BitcoinTransaction),
```

Comment:
You can't construct a `TaprootSpendInfo` for a script spend unless you know all the leaves of the script tree.  My understanding of commit/reveal is that we only know how to spend one of the paths.

We can get around this if we specify that the reveal is actually an internal key spend with the aggregate key as the internal key.

Reply:
If we aren't going to use an internal key spend then we need a `ControlBlock` not a `TaprootSpendInfo`.